modules/social_features/social_content_report/social_content_report.install

```diff
@@ -88,3 +92,16 @@ function social_content_report_install() {
 function social_content_report_update_last_removed() : int {
   return 11401;
 }
+
+/**
+ * Grunt permission to see/edit/delete report_reasons vocabulary terms for SM.
+ */
+function social_content_report_update_130001(): void {
+    $permissions = [
+        'access taxonomy overview',
+        'create terms in report_reasons',
+        'delete terms in report_reasons',
+        'edit terms in report_reasons',
+    ];
+    user_role_grant_permissions('sitemanager', $permissions);
+}
```

Comment:
@Macsym2018 Add empty line in the end here and in other places

Reply:
@ribel Added